### PR TITLE
(packaging) Bump shared rubygem versions

### PR DESCRIPTION
**Pull request by lucywyman**
This updates rubygems used by multiple projects to their latest versions.

@puppetlabs/night-s-watch Need approval for bumping all three gems

@puppetlabs/installer-and-management Need approval for `ffi`

@puppetlabs/pdk Need approval for `ffi`

### CODE REVIEW

**Comment by scotje** [comment]
Going to build a `pdk-runtime` package from this and then run it through our tests.

**Comment by gimmyxd** [comment]
Jenkins please test this with puppet-agent master on aix61-POWERfa,solaris10-64a,windows2012r2-64a

File: configs/components/rubygem-concurrent-ruby.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 component 'rubygem-concurrent-ruby' do |pkg, settings, platform|
-  pkg.version '1.1.5'
-  pkg.md5sum '4409c2d6925d8448cb34a947eacaa29b'
+  pkg.version '1.1.6'
```
Comment (donoghuc) [inline configs/components/rubygem-concurrent-ruby.rb:2]: I think i tried bumping this before and hit https://github.com/ruby-concurrency/concurrent-ruby/pull/856#issuecomment-604124810 I verified that patch stops the segfault, but it has not been merged/released yet. I think we should wait for that to be addressed.

Comment (lucywyman) [reply configs/components/rubygem-concurrent-ruby.rb]: Removed!

**Comment by lucywyman** [comment]
Just as a heads up, PDK had trouble verifying a build due to a Litmus dependency failing to compile. We're waiting on that issue to get resolved so that they can ensure this works.

**Pull request merged**